perllib/FixMyStreet/ImageMagick.pm

```diff
@@ -0,0 +1,68 @@
+package FixMyStreet::ImageMagick;
+
+use Moo;
+
+my $IM = eval {
+    require Image::Magick;
+    Image::Magick->import;
+    1;
+};
+
+has blob => ( is => 'ro' );
+
+has image => (
+    is => 'rwp',
+    lazy => 1,
+    default => sub {
+        my $self = shift;
+        return unless $IM;
+        my $image = Image::Magick->new;
+        $image->BlobToImage($self->blob);
+        return $image;
+    },
+);
+
+sub strip {
+    my $self = shift;
+    return $self unless $self->image;
+    $self->image->Strip();
+    return $self;
+}
+
+sub rotate {
+    my ($self, $direction) = @_;
+    return $self unless $self->image;
+    my $err = $self->image->Rotate($direction);
+    return 0 if $err;
+    return $self;
+}
+
+# Shrinks a picture to the specified size, but keeping in proportion.
+sub shrink {
+    my ($self, $size) = shift;
```

Comment:
Typo:
```suggestion
    my ($self, $size) = @_;
```

This was a fun one to track down - nothing was breaking! The result was full-size images being output as e.g. `web/photo/1432427.0.jpeg` because no new size was being specified.